packages/composite-checkout/src/components/checkout.js

```diff
@@ -133,7 +134,7 @@ function OrderSummaryStep( { CheckoutHeader } ) {
 			isComplete={ true }
 			stepNumber={ 0 }
 			title={ localize( 'You are all set to check out' ) }
-			stepSummary={ <OrderSummaryContent CheckoutHeader={ CheckoutHeader } /> }
+			stepSummary={ <CheckoutOrderSummary CheckoutHeader={ CheckoutHeader } /> }
```

Comment:
oh right. Good call. In addition to the CheckoutSummary, what do you think about making the Coupon a feature the host can turn on/off? So we'd essentially also create a `enableCoupon` prop that you pass to demo/index.js's `Checkout` component?

Reply:
or do you think it's better to include the coupons in the WPcom specific components?

Reply:
Right now I think that the entire coupon behavior should live in WPCom-specific components. The whole concept of a coupon might vary based on the host so including it in any default components could add a lot of complexity in order to make it flexible enough. We _can_ make the WPCom version flexible within itself, though! That way it can maybe be reused for different hosts. So for example, if you wanted to add a `couponsEnabled` prop, I'd add it to the `WPComOrderSummary` component (and whatever other wpcom components might need it), not the `Checkout` component.

Reply:
ok, that makes sense. I'll leave the extra prop out for now because probably don't need it right now. We can always add it later if we have a concrete need.